contracts/simple_dex/lib.rs

```diff
@@ -322,6 +322,11 @@ mod simple_dex {
             Ok(())
         }
 
+        #[ink(message)]
+        pub fn can_swap_pair(&self, from: AccountId, to: AccountId) -> bool {
```

Comment:
Please add documentation on the public endpoint. It gets compiled into `metadata.json` file and may be useful for frontend wallets like Contracts UI.

Reply:
done